CMakeLists.txt

```
@@ -157,13 +156,9 @@ endif()
 
 set (PTHREAD_LIB pthread)
 
-if (ENABLE_LLD AND ENABLE_MOLD)
-  message (FATAL_ERROR "Cannot enable both lld and mold")
-elseif (ENABLE_LLD)
-  set (LD_OVERRIDE "-fuse-ld=lld")
-  add_link_options(${LD_OVERRIDE})
-elseif (ENABLE_MOLD)
-  add_link_options("-fuse-ld=mold")
+if (NOT USE_LINKER STREQUAL "")
```

Empty strings evaluate to false already:

```suggestion
if (USE_LINKER)
```

---

resolved.